Libraries/MSSQLDevice/Source/Device.MSSQL/MSSQLTypes.cs

```diff
@@ -446,11 +450,83 @@ public class MSSQLBinary : SQLBinary
 
 		protected override string InternalNativeDomainName(MetaData AMetaData)
 		{
-			return "image";
+			return "varbinary(max)";

```

Comment:
You are correct, it is not required.  I should be done as an independent, potentially breaking change.  It is a good change though.  I've had to work around the current blob type mappings a few times.